source/v4/users.yaml

```diff
@@ -485,3 +485,70 @@
           $ref: '#/responses/Unauthorized'
         '404':
           $ref: '#/responses/NotFound'
+
+  /users/{user_id}/sessions:
+    get:
+      tags:
+        - users
+      summary: Get a list of sessions by User GUID.
```

Comment:
Can we simplify the summary to just be `Get a user's sessions`? It'll be showing up in a sidebar so it's better if we can keep the summaries short :)